draft-ietf-scitt-architecture.md

```diff
@@ -367,43 +367,51 @@ This section describes at a high level, the three main roles and associated proc
 
 ## Transparency Service
 
-An important function of a Transparency Service is to maintain Registration Policies for the Append-only Log.
-The Append-only Log is the verifiable data structure which registers Signed Statements and supports the production of Receipts.
+Transparency Services MUST feature an Append-only Log. The Append-only Log is the verifiable data structure which registers Signed Statements and supports the production of Receipts.
```

Comment:
not to challenge our long standing tradition of using the term "Append-only log", but technically the Receipt depends on a verifiable data structure, of which an append only log (such as a merkle tree), is just one form.